update-sdks.sh

```diff
@@ -1,23 +0,0 @@
-#!/bin/bash
```

Comment:
We can clean up the RELEASING.md a little bit too 🎊

Reply:
good catch. I have an android device set to arrive today, when it gets here I'll do a pass of testing and I'll clean up RELEASING.md

Reply:
coming back to this, what _should_ be updated in RELEASING.md?

Reply:
@vegaro I'm removing the hold and merging for now and I can follow up with any needed changes to RELEASING.md in a separate PR